plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckAccuracyRadiusAnimator.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.maps.plugin.locationcomponent.animators
+
+import android.graphics.Color
+import androidx.annotation.ColorInt
+import com.mapbox.maps.plugin.locationcomponent.OnIndicatorAccuracyRadiusChangedListener
+
+internal class PuckAccuracyRadiusAnimator(private val accuracyRadiusChangedListener: OnIndicatorAccuracyRadiusChangedListener) :
+  PuckAnimator<Double>(Evaluators.DOUBLE) {
+
+  var enabled = false
+  @ColorInt
+  var accuracyCircleColor = Color.BLUE
+  var accuracyCircleAlpha = 1.0f
+
+  override fun updateLayer(fraction: Float, value: Double) {
+    if (enabled) {
+      locationRenderer?.setAccuracyRadius(value.toFloat())
```

Comment:
should we set only positive values?
```suggestion
      locationRenderer?.setAccuracyRadius(max(0f, value.toFloat()))
```

Reply:
Yes, that would be better.